00_Core/Scripts/Source/MilkMCM.psc

```diff
@@ -3231,7 +3231,7 @@ endState
 state Debug_MM_MilkMax_Basevalue_Slider
 	event OnSliderOpenST()
 		SetSliderDialogStartValue(MME_Storage.getMilkMaxBasevalue(MaidlistA[MaidIndex]))
-		SetSliderDialogDefaultValue(0)
+		SetSliderDialogDefaultValue(2)
 		SetSliderDialogRange(0, 20)
```

Comment:
maybe i dont understand what it does or how it works, but maximum milk should never be less than 1